_articles/de/building-community.md

```diff
@@ -14,97 +14,107 @@ related:
 
-* **Make it easy for someone to use your project.** [A friendly README](../starting-a-project/#writing-a-readme) and clear code examples will make it easier for anyone who lands on your project to get started.
-* **Clearly explain how to contribute**, using [your CONTRIBUTING file](../starting-a-project/#writing-your-contributing-guidelines) and keeping your issues up-to-date.
+* **Helfen Sie beim Einstieg in ihr Projekt.** [eine freundliche README](../starting-a-project/#writing-a-readme) und anschauliche Codebeispiele erleichtern allen, die auf Ihr Projekt stoßen, den Einstieg.
+* **Erklären Sie in klaren Worten, wie Leute beitragen sollen**, bspw. in [einer CONTRIBUTING-Datei](../starting-a-project/#writing-your-contributing-guidelines), und indem Sie die Issues stets aktuell halten.
 
-[GitHub's 2017 Open Source Survey](http://opensourcesurvey.org/2017/) showed incomplete or confusing documentation is the biggest problem for open source users. Good documentation invites people to interact with your project. Eventually, someone will open an issue or pull request. Use these interactions as opportunities to move them down the funnel.
+[GitHubs 2017er Open-Source-Umfrage](http://opensourcesurvey.org/2017/) zeigte, dass unvollständige oder verwirrende Dokumentation das größte Problem für Open-Source-Anwender\*innen ist. Eine gute Dokumentation lädt dazu ein, mit Ihrem Projekt zu interagieren. Wenn jemand ein Issue oder einen Pull-Request erstellt, haben Sie die Möglichkeit, diese Person durch den Trichter zu bugsieren.
 
-* **When someone new lands on your project, thank them for their interest!** It only takes one negative experience to make someone not want to come back.
-* **Be responsive.** If you don't respond to their issue for a month, chances are, they've already forgotten about your project.
-* **Be open-minded about the types of contributions you'll accept.** Many contributors start with a bug report or small fix. There are [many ways to contribute](../how-to-contribute/#what-it-means-to-contribute) to a project. Let people help how they want to help.
-* **If there's a contribution you disagree with,** thank them for their idea and [explain why](../best-practices/#learning-to-say-no) it doesn't fit into the scope of the project, linking to relevant documentation if you have it.
+* **Wenn Leute neu auf Ihrem Projekt landen, danken Sie ihnen für ihr Interesse!** Es braucht nur eine negative Erfahrung, um jemanden nachhaltig abzuschrecken.
+* **Antworten Sie schnell.** Wenn Sie einen Monat lang nicht auf jemandes Issue reagieren, ist es wahrscheinlich, dass diese Person Ihr Projekt bereits vergessen hat.
+* **Nehmen Sie verschiedener Beitragsarten an.** Viele Mitwirkende beginnen mit einem Fehlerbericht oder einer kleinen Korrektur: Es gibt [viele Möglichkeiten, einen Beitrag zu leisten](../how-to-contribute/#what-it-means-to-contribute) zu einem Projekt. Lassen Sie Leute so helfen, wie diese es möchten.
```

Comment:
add you an todo to keep the the link working

Reply:
Do you mean in case we rename the files in the end? I think I won't do that after all, see https://github.com/github/opensource.guide/pull/577 > "Renaming the files"

Reply:
yep.